swagger.json

```diff
@@ -731,10 +731,7 @@
           },
           "product": {
             "description": "A reference to the product record. This field is only required if the description for this line is not passed in.",
-            "oneOf": [
-              { "$ref": "#/components/schemas/ExternalId" },
-              { "$ref": "#/components/schemas/Product" }
-            ]
+            "allOf": [{ "$ref": "#/components/schemas/Product" }]
```

Comment:
is this InvoiceLine model reused on the ougoing request? if so this change will give us trouble.

Reply:
we are always giving full invoice information so I don't think so